src/timeline/timeline.style.tsx

```diff
@@ -3,6 +3,7 @@ import { Color } from "../color";
 import { MediaQuery } from "../media";
 import { Text } from "../text";
 import { TimelineStatusProps, Variant } from "./types";
+import { ExclamationCircleFillIcon } from "@lifesg/react-icons/exclamation-circle-fill";
```

Comment:
Reorder imports

Reply:
I've removed this import and migrate the styling of `ExclamationCircleFillIcon` to `CircleIndicator`.